db/schema.rb

```diff
@@ -111,6 +112,19 @@
     t.index ["name"], name: "index_coalitions_on_name", unique: true
   end
 
+  create_table "consents", force: :cascade do |t|
```

Comment:
I think you've got a dirty DB from the other ticket you were working on! Might need to drop your db, or can just manaully remove this from the schema.

Reply:
good catch!!